### PR TITLE
Update timestamp precision to match MySQL/Liquibase

### PR DESCRIPTION
**Pull request by jmthibault79**
Not for a specific JIRA story but may help with GAWB-418.

Makes testing easier for GAWB-715 as well.

No Liquibase change here because it is already correct.
- [x] **Submitter**: Rebase to develop. DO NOT SQUASH
- [x] **Submitter**: Make sure Swagger is updated if API changes
- [x] **Submitter**: Make sure documentation for code is complete
- [x] **Submitter**: Make sure liquibase is updated if appropriate
- [x] **Submitter**: Review code comments; remove done TODOs, create stories for remaining TODOs
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Add description or comments on the PR explaining the hows/whys (if not obvious)
- [x] Tell  that the PR exists if he wants to look at it **(apply requires_doge label)**
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
- [x] **LR**: Initial review by LR and others.
- [x] Comment / review / update cycle:
  - Rest of team may comments on PR at will
  - **LR assigns to submitter** for feedback fixes
  - Submitter updates documentation as needed
  - Submitter rebases to develop again if necessary
  - Submitter makes further commits. DO NOT SQUASH. **Reassign to LR** for further feedback
- [x]  sign off
- [x] **LR** sign off
- [x] **Assign to submitter** to finalize
- [x] **Submitter**: Squash commits, rebase if necessary
- [x] **Submitter**: Verify all tests go green, including CI tests
- [x] **Submitter**: Merge to develop 
- [x] **Submitter**: Delete branch after merge
- [x] **Submitter**: Check configuration files in Jenkins in case they need changes
- [x] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [x] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [x] **Submitter**: Inform other teams of any API changes via hipchat and/or email
- [x] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/TestDriverComponent.scala [hunk 1]
```diff
@@ -39,10 +39,9 @@ trait TestDriverComponent extends DriverComponent with DataAccess {
   val testDate = currentTime()
   val userInfo = UserInfo("test_token", OAuth2BearerToken("token"), 123, "123456789876543212345")
 
-  // NOTE: we are setting the millis here to 0 to match the truncation from the database - if we switch to using liquibase for tests, we may need to revert this
-  def currentTime(): DateTime = {
-    new DateTime().withMillisOfSecond(0)
```
Comment (jmthibault79) [inline src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/TestDriverComponent.scala:44]: This made testing GAWB-715 more difficult.

**Comment by MatthewBemis** [comment]
👍

**Comment by davidangb** [comment]
Thank you for this!!! [SubmissionApiServiceSpec line 177](https://github.com/broadinstitute/rawls/blob/develop/src/test/scala/org/broadinstitute/dsde/rawls/webservice/SubmissionApiServiceSpec.scala#L177) is a great place to test this.

**Comment by dvoet** [comment]
👍

**Comment by jmthibault79** [comment]
@davidangb I updated that line.  It fails without this change, and passes with it.  Thanks!

**Pull request merged**